### PR TITLE
Improved instructions on how to install GHC

### PR DESCRIPTION
**Pull request by newhoggy**
Includes clarification on GHC version to use

### CODE REVIEW

**Comment by newhoggy** [comment]
This PR is in response to https://github.com/input-output-hk/cardano-node/pull/3118 and supersedes that PR.

**Review by kevinhammond — APPROVED** [review]
Checked and looks good

File: doc/getting-started/install.md [hunk 1]
```diff
@@ -75,21 +75,22 @@ Confirm that you installed cabal version `3.4.0.0`:
...
-Create a working directory for your builds:
-
-    mkdir -p ~/src
-    cd ~/src
+Download and install version `8.10.4` of GHC.  The easiest way to do this is to use [ghcup](https://www.haskell.org/ghcup/).
 
- Download and install the latest version of GHC:
+```bash
+$ curl --proto '=https' --tlsv1.2 -sSf https://get-ghcup.haskell.org | sh
+$ ghcup install ghc 8.10.4
```
Comment (rdlrt) [inline doc/getting-started/install.md:82]: You might need to source the bashrc to set the env variables

Comment (newhoggy) [reply doc/getting-started/install.md:87]: Good point.  I've updated the instructions.

**Comment by newhoggy** [comment]
bors merge

**Pull request merged**